src/util/getTokensInRange.ts

```diff
@@ -12,7 +12,7 @@ export function getTokensInRange(
   editor: vscode.TextEditor,
   range: vscode.Range
 ): PartialToken[] {
-  const text = editor.document.getText(range).toLowerCase();
+  const text = editor.document.getText(range);
```

Comment:
This function really shouldn't have been lower-casing, as it is supposed to be a generic function

Reply:
Definitely